playbooks/unmount-and-clear-disks.yml

```diff
@@ -0,0 +1,30 @@
+
+---
+- name: unmount and clear all drives
+  hosts: "{{ hosts }}"
+
+  tasks:
+  - name: install dependencies
+    apt: pkg="gdisk"
+
+  - set_fact: disk_list={{ swift.disks }}
+    when: hosts == "swiftnode"
```

Comment:
Is this not intended to run on anything but swiftnodes ? If so, do a fail task above that will fail if the host isn't in the `swiftnode` group. Then you can assume the rest of the tasks are on a swiftnode.